package.json

```diff
@@ -34,15 +34,15 @@
     "items": "2.x.x"
   },
   "peerDependencies": {
-    "hapi": ">=10 <16"
+    "hapi": ">=10 <17"
```

Comment:
If this doesn't work yet on hapi v16, then no need for this change.

Reply:
oh bugger, extra sloppy. sorry about that, thanks for catching 🙏